tests/test_version.py

```diff
@@ -5,6 +5,21 @@
 import argparse
 import zenodopy
 
+def parse_metadata_from_json(json_file_path: Path) -> zenodopy.ZenodoMetadata:
+    """Parse metadata from a JSON file into a ZenodoMetadata object."""
```

Comment:
Why not make this a static method of the `ZenodoMetadata` so that others can also use this? Unless it is specific to this test, and can't be used elsewhere?